### DISCUSSION TITLE
PWA manifest and Nuxt manifest have conflict.

### DISCUSSION THREAD

**Discussion by mjzarrin** [nuxt/nuxt#16759] [Questions]
I could not run PWA on production mode because PWA manifest get replaced by Nuxt manifest. 
is there any way to change the name of Nuxt manifest?

**Comment by palamarchukser** [comment]
👋 @mjzarrin 
do you need a generated by nux manifest? just disable it
`const config = {
  generate: {
    manifest: false
  },
}`

**Comment by danielroe** [comment]
This is no longer the case; we don't emit a Nuxt client manifest: https://github.com/nuxt/framework/pull/7021.

**Reply by marekmis** [reply]
How can I enable generating Nuxt client manifest?

I was relying on Nuxt client manifest for my app version control - as that was the file which file name never changed with consecutive builds (it's a static site using generate). App was fetching the manifest file every X seconds to check for changes in file, and if detected a page reload would occur.

**Reply by marekmis** [reply]
Solved my problem by writing a custom plugin for Vite, here it is in case anyone needs some sort of version number/timestamp trace in files:

```
// nuxt.config.js

vite: {
    build: {
      rollupOptions: {
        plugins: [
          {
            name: 'BuildVersion',
            generateBundle() {
                this.emitFile({
                  type: 'asset',
                  fileName: 'my-file.json',
                  source: JSON.stringify({'BUILD_DATE' : new Date().toISOString()})
                });
            }
          }
        ]
      }
    }
}
```
